dynamic-layers/qt5-layer/recipes-qt/images/core-image-base.bbappend

```diff
@@ -2,4 +2,5 @@ inherit populate_sdk_qt5
 
 IMAGE_INSTALL_append = " \
 	packagegroup-qt5 \
+	packagegroup-fonts-truetype-core \
```

Comment:
This commit adds a dependency to the meta-openembedded.
Can you try with `liberation-fonts` instead ?

Reply:
Yes it's works too i push with this ;)